projects/m2/016-reverse-lookup/js/index.js

```diff
@@ -0,0 +1,31 @@
+
+
+
+
+
+function reverseLookup(dictionary, value){
+    const keys = Object.keys(dictionary);
+    const foundKeys = [];
+    for (let i = 0; i < keys.length; i++) {
+        const key = keys[i];
+        if (dictionary[key] === value) {
+            foundKeys.push(key)
+        }
```

Comment:
Your solution is correct. But it can be **improved**.
Try solving using the `entries` method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries